test/unit/redux/studio-member-actions.test.js

```diff
@@ -406,26 +406,87 @@ describe('acceptInvitation', () => {
         expect(state.studio.invited).toBe(true);
         expect(state.studio.curator).toBe(false);
     });
+});
```

Comment:
Was this just incorrectly missing before? I don't see a corresponding change that adds another "describe"

Reply:
I'm not totally sure I understand what you mean by another "describe." But previously, the 'transferHost' tests were incorrectly nested under the 'promoteCurator' tests. I moved them out. (line 411)

Reply:
Cool! That's exactly what I was asking.